src/SprykerEco/Yves/Heidelpay/Theme/default/components/molecules/direct-debit-existing-registration/direct-debit-existing-registration.twig

```diff
@@ -0,0 +1,25 @@
+{% extends model('component') %}
+
+{% define config = {
+    name: 'direct-debit-existing-registration',
+} %}
+
+{% define data = {
+    paymentOption: data.paymentOption,
+    paymentMethod: data.paymentMethod,
+} %}
+
+{% block body %}
+    {{ form_widget(data.paymentOption) }}
+    {{ form_label(data.paymentOption) }}
+    <div class="{{ config.name }}__option spacing-top">
+        <small>
```

Comment:
Why do you need an additional tag here? Why not 
```
<small class="{{ config.name }}__option spacing-top">
```
or
```
<div class="{{ config.name }}__option spacing-top text-small">
```

Reply:
fixed